packages/elements/src/item/index.ts

```diff
@@ -215,14 +274,22 @@ export class Item extends ControlElement {
    * Get subLabel template if it is defined and no slot content present
    */
   private get subLabelTemplate (): TemplateResult | undefined {
-    return this.subLabel && this.isSlotEmpty ? html`<div part="sub-label">${this.subLabel}</div>` : undefined;
+    return html`<div part="sub-label" ${ref(this.subLabelRef)}>${this.subLabel}</div>`;
   }
 
   /**
    * Get label template if it is defined and no slot content present
    */
   private get labelTemplate (): TemplateResult | undefined {
-    return this.label && this.isSlotEmpty ? html`${this.label}` : undefined;
+    return html`${this.label}`;
+  }
+
+  /**
+   * Get slot content
+   */
+  private get slotContent (): string {
+    const slot = this.slotRef.value as HTMLSlotElement;
+    return slot.assignedNodes().map(e => e.textContent).join(' ').trim();
```

Comment:
You're still casting type here. We should make sure the slot is available.
```suggestion
    const nodes = this.slotRef.value?.assignedNodes() || [];
    return nodes.map(e => e.textContent).join(' ').trim();
```

Reply:
So embarrassed to me 🫣, Thanks! Trem.

Reply:
Updated.